client/src/app/components/forms/rsvp/rsvp.component.ts

```diff
@@ -89,4 +105,38 @@ export class RsvpFormComponent {
 
 		this.submit.emit(rsvpData);
 	}
+
+	populateForm() {
+		if (this.existingRsvpData) {
```

Comment:
I think you're checking against null here? In that case, I would just use nullish coalescing operators for this bit.

Something like

```
const firstName: string = this.existingRsvpData.firstName ?? ''; 
const lastName: string = this.existingRsvpData.lastName ?? ''; 
...

this.rsvpForm.controls.firstName.setValue(firstName);
this.rsvpForm.controls.firstName.setValue(lastName);
...
```

Something like that